db/migrate/20150427224114_add_o_auth_to_app.rb

```diff
@@ -0,0 +1,9 @@
+class AddOAuthToApp < ActiveRecord::Migration
+  def change
+    add_column :apps, :redirect_uri, :string
+    add_column :apps, :homepage, :string
+    add_column :apps, :description, :string
+    add_column :apps, :privileged, :boolean, null: false, default: false
```

Comment:
I don't think there's a need. It's not like it's something these apps pay for, it's just a bit of increased scrutiny. There's no more ranks to it except vetted and unvetted. If circumstances change, a migration is easy, so there's no need to think that far ahead